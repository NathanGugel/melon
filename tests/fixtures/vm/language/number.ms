func testNumber |i| => {
    io.print(number.fromString(i));
}

testNumber("5e10");
testNumber("5e-5");
testNumber("5.456e3");
testNumber("5asdasd");

io.print(10 * 15);
io.print(10 * 15.0);
io.print(10.0 * 15);