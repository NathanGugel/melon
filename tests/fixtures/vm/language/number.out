50000000000.000000000
0.000050000
5456.000000000
null
150
150.000000000
150.000000000
